package.json

```diff
@@ -33,6 +33,7 @@
     "bn.js": "4.11.6",
     "date-fns": "2.0.0-alpha.22",
     "eth-provider": "^0.2.0",
+    "ethereum-ens": "^0.7.7",
```

Comment:
This library alone is almost [0.5 MB](https://bundlephobia.com/result?p=ethereum-ens@0.7.7) gzipped :scream: Should we try to use the exact same version than the one in `@aragon/wrapper` until we remove it there, and maybe try to deduplicate it from the final build?

Reply:
Okey... will migrate back to using `ethjs-ens`. @jefflau has put up [an experimental lightweight version](https://github.com/jefflau/ens-lite) that we can experiment with, but we should do this migration later.